docs/data/toolpad/core/introduction/tutorial.md

```diff
@@ -1,3 +1,105 @@
-# Tutorial
+---
+title: Tutorial
+---
 
-<p class="description">Tutorial</p>
+# Toolpad Core - Tutorial
+
+<p class="description">Learn how to use Toolpad Core through an illustrative example dashboard.</p>
+
+## Bootstrapping
+
+<br/>
+
+1. To choose a project name and create a basic project for this tutorial, run:
+
+<codeblock storageKey="package-manager">
+
+```bash npm
+npx create-toolpad-app@latest --example core-tutorial
+```
+
+```bash pnpm
+pnpm create toolpad-app --example core-tutorial
+```
+
+```bash yarn
+yarn create toolpad-app --example core-tutorial
+```
+
+  </codeblock>
+
+2. To start the basic project on [http://localhost:3000](http://localhost:3000/), run:
+
+<codeblock storageKey="package-manager">
+
+```bash npm
+cd <project-name>
+npm install && npm run dev
+```
+
+```bash pnpm
+cd <project-name>
+pnpm install && pnpm dev
+```
+
+```bash yarn
+cd <project-name>
+yarn && yarn dev
+```
+
+</codeblock>
+
+3. The following splash screen appears:
+
+{{"component": "modules/components/DocsImage.tsx", "src": "/static/toolpad/docs/core/installation-1.png", "alt": "Toolpad Core entry point", "caption": "Starting with Toolpad Core", "zoom": true, "indent": 1 }}
+
+4. The app has one page already created, which can be viewed by clicking on the "Go to page" button. The following page should appear:
+
+{{"component": "modules/components/DocsImage.tsx", "src": "/static/toolpad/docs/core/tutorial-1.png", "alt": "Toolpad Core default page", "caption": "Default page in dashboard layout", "zoom": true, "indent": 1 }}
```

Comment:
Can we replace this with a live demo of what you'd see? The demo doesn't have to actually be running a Next.js app, we can hide the source code.